client/landing/jetpack-cloud/components/scan-threats/index.tsx

```diff
@@ -51,11 +51,9 @@ const ScanError: React.FC< { site: Site } > = ( { site } ) => {
 
 	return (
 		<div className="scan-threats__error">
-			{ translate( 'The scanner was unable to check all files and errored before completion.' ) }
-			<br />
 			{ isEnabled( 'jetpack-cloud/on-demand-scan' )
 				? translate(
-						'Deal with the threats found above and run the {{runScan}}scan again{{/runScan}}. If the error persists, we are {{linkToSupport}}here to help{{/linkToSupport}}.',
+						'The scanner was unable to check all files and errored before completion. Deal with the threats found above and run the {{runScan}}scan again{{/runScan}}. If the error persists, we are {{linkToSupport}}here to help{{/linkToSupport}}.',
```

Comment:
:warning: This change will be queued for retranslation. We'll display the English text until then.